pkg/addons/translate.go

```diff
@@ -343,19 +356,30 @@ var Translations = map[string][]struct {
 	},
 	"Secret/kube-service-catalog/apiserver-ssl": {
 		{
-			Path:     jsonpath.MustCompile("$.stringData.'tls.crt'"),
+			Path:     jsonpath.MustCompile("$.stringData.'apiserver.crt'"),
```

Comment:
no - the single quotes are required because of the period.  we translate the Secret "data" to "stringData" at https://github.com/openshift/openshift-azure/blob/master/pkg/addons/clean.go#L45 to save extra hassle with base64 encodings.